static/files/alpha4_download-data-for-wlan-firmwares.txt

```diff
@@ -1,56 +0,0 @@
-#!/bin/sh
```

Comment:
This file wasn't referenced on the website as far as I could tell and none of the links seemed to work or were duplicated by the "download-data-for-wan-firmwares.txt".

Reply:
Thir was for release Alpha 4 (before package management existed), it's ok to delete it now.